target/linux/generic/files/drivers/platform/mikrotik/Kconfig

```diff
@@ -12,6 +10,9 @@ if MIKROTIK
 
 config MIKROTIK_RB_SYSFS
 	tristate "RouterBoot sysfs support"
+	depends on MTD
+	select LZO_DECOMPRESS
+	select CRC32
```

Comment:
I'm actually wondering if I shouldn't force select 4K_SECTORS here too, to be on the safe side. Or change the code to make it read-only if 4K_SECTORS isn't compiled-in. Preferences?

Reply:
IMO the mtd layer should do the right thing when EB is too large for the target size regardless of 4K_SECTORS, but it's unclear to me that it does. Maybe a question for nbd?

Reply:
@nbd168 
Can you kindly comment on the MTD stuff?

It's one of the last things holding this PR back.

Appreciated!

Reply:
I opted to make the driver act read-only if 4K_SECTORS aren't enabled. It still won't handle the case where (if) partial erase is broken though.
Rationale: it may be useful to access these values read-only even if they can't be modified, and it seems overkill to force enable 4K_SECTORS from this driver alone, the choice should be left to the build.